app/src/main/java/com/android/sample/ui/activitydetails/ActivityDetailsScreen.kt

```diff
@@ -260,6 +263,16 @@ fun ActivityDetailsScreen(
                                   "${round(distance * 10) / 10}km"
                                 }
                         Text(text = distanceString, modifier = Modifier.testTag("distanceText"))
+                        Spacer(modifier = Modifier.height(STANDARD_PADDING.dp))
+                        // text field button to navigate to the activity's location on the map
+                        // screen
+                        Text(
+                            text = stringResource(id = R.string.button_to_map),
+                            modifier =
+                                Modifier.testTag("activityToMapText")
+                                    .clickable(
+                                        onClick = { navigationActions.navigateTo(Screen.MAP) }),
+                            style = TextStyle(textDecoration = TextDecoration.Underline))
```

Comment:
Good for the string that was added in the string.xml file and also for having chosen a good text style.

Reply:
this test Tag haven't been tested.

Reply:
Yes, I have warned you privately on this. Because of some technical issues and because I had not much time left (a lot of merges waiting on pipeline) I decided to leave this for next week.